rfcs/2023-11-08-font-loading-strategy.md

```diff
@@ -195,6 +195,16 @@ Cons from all of the above 🙈
 
 ## Implementation Details
 
+### Font Size Optimization
+
+To optimize on font-size, we are splitting fonts into multiple files.
+
+- inter-latin-rzp.woff2 (alphabets, numbers, common symbols, currency)
```

Comment:
can we name them as blade rather than rzp?

Reply:
yup better